src/main/java/uk/gov/hmcts/reform/sscs/model/hearings/UnavailabilityDayOfWeek.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.sscs.model.hearings;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class UnavailabilityDayOfWeek {
+
+    @JsonProperty("DOW")
+    private String dayOfWeek;
+
+    @JsonProperty("DOWUnavailabilityType")
+    private String dayOfWeekUnavailabilityType;
```

Comment:
Would this be better as Enum?

Reply:
I had Enum but since we not having validation so I have remove them as this will get value from similar source